patches/chrome-browser-ui-messages-android-java-src-org-chromium-chrome-browser-ui-messages-snackbar-SnackbarView.java.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/chrome/browser/ui/messages/android/java/src/org/chromium/chrome/browser/ui/messages/snackbar/SnackbarView.java b/chrome/browser/ui/messages/android/java/src/org/chromium/chrome/browser/ui/messages/snackbar/SnackbarView.java
+index 1a1ca7cd94ef458e72706dcb5251e21dd5c6ccc8..2ccb5d5a71eb2cb53b00144c2781140024582cf2 100644
+--- a/chrome/browser/ui/messages/android/java/src/org/chromium/chrome/browser/ui/messages/snackbar/SnackbarView.java
++++ b/chrome/browser/ui/messages/android/java/src/org/chromium/chrome/browser/ui/messages/snackbar/SnackbarView.java
+@@ -99,6 +99,7 @@ public class SnackbarView implements InsetObserver.WindowInsetObserver {
+      * @param windowAndroid The WindowAndroid used for starting animation. If it is null,
+      *     Animator#start is called instead.
+      */
++    /*
+     public SnackbarView(
```

Comment:
What are the troubles with this c-tor?

Reply:
as per DM I removed that patch and made a direct patching of `new SnackbarView`